chatterLink-frontend/src/Components/Chats/ChatComp.jsx

```
@@ -0,0 +1,40 @@
+    //console.log("Chatmmmessage:", chatMessages)
+    return (
+        <div className="chat-data-container">
+            {
+                chatMessages?.length > 0 ? (
+                    <div className="chat-messages-data">
+                        {
+                            chatMessages.map((message) => (
+                                <ChatBubble
+                                    message={message.content}
+                                    isSender={message.senderId === currentUser.userId}
+                                 />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)